templates/job/docs/__skillname__/README.md

```diff
@@ -4,6 +4,22 @@ This workspace template demonstrates a simple job skill skeleton that simply pri
 
 ## Building and Publishing the Skill
 
-In order to build the job skill Docker image, simply select "Build Skills" from the Cortex menu in the IDE.  You can also right-click on the `skill.yaml` file for the skill and select "Build Skill" from the Cortex context menu.
+### Using the cortex cli
+
+Building skill(s)
+
+```
+cortex workspaces build
+```
+
+Publishing  skill and docker image
+
+```
+cortex workspaces publish
```

Comment:
To publish all docker images and skills in the workspace:
```
cortex workspaces publish
```

To publish all docker images and skills in the folder:
```
cortex workspaces publish [folder]
```

To publish only the specified skill and its docker image:
```
cortex workspaces publish --skill <skill name>
```